packages/x-date-pickers-pro/src/DateRangePicker/DateRangePickerToolbar.tsx

```diff
@@ -102,6 +103,7 @@ const DateRangePickerToolbar = React.forwardRef(function DateRangePickerToolbar<
       className={clsx(className, classes.root)}
       ownerState={ownerState}
       ref={ref}
+      {...other}
```

Comment:
We probably want to put the `isLandscape={false}` after the spread if we want to keep forcing the orientation (or check if it works correctly in landscape now if we want to change the behavior).

Reply:
Would you say that it is not working correctly? 🤔 
<img width="810" alt="Screenshot 2023-01-16 at 16 46 05" src="https://user-images.githubusercontent.com/4941090/212729283-5c924cc2-9241-4908-8e90-72c7cdcbebe3.png">

Reply:
The main usage of `isLandscape` is probably the lanscape mode on mobile (without any customization from the user).
And I guess it should be working fine.

And for the manual landscape on desktop, as your screen shows, it seems to work fine.

Just wanted to notice that it was a behavior change.

Reply:
Thanks for noting. Did a deeper comparison and noticed that it does introduce a visual regression.
Changed the spread to be at the top until the toolbar layout is fixed. 👌